_posts/2018-10-09-how-we-work-pair-programming.md

```diff
@@ -0,0 +1,110 @@
+# How We Work: Pair Programming
+
+by Jason Fry (@JasonMFry)
+
+![A photo of two people working at a pairing station][]
+use real photo, not this one.
+
+## Why We Pair
+
+1. Improves our Team
+    1. It forces us to become more personable, patient, and articulate
+    1. It makes junior engineers and new hires more productive faster
+    1. Spreads knowledge
+    1. Improves our cohesion as a team
+1. Improves our code
+    1. Two people make better design decisions than one person
+    1. Part of the Navigator's job is to spot opportunites for refactoring, so we're constantly
```

Comment:
The term "navigator" hasn't been introduced by this point, which may or may not be a problem. Maybe we could say that we're constantly doing small refactorings instead, and then expand on that in the pairing style section? Or maybe it's fine as it is, I dunno.